### PR TITLE
OMERO.server 5.6.5 components

### PR DESCRIPTION
**Pull request by sbesson**
- Bumps the OMERO.server Java components to the latest released versions
- add changelog entry including more information about the changes

### CODE REVIEW

**Review by jburel — DISMISSED** [review]

File: history.rst [hunk 1]
```diff
@@ -5,6 +5,35 @@
...
+- omero-romio 5.7.0
+- omero-common 5.5.10
+- omero-model 5.6.7
+
+
+Improvements include:
+
+-  a new server configuration allowing to control the pyramidal requirement for floating-point images
+-  an upgrade of Bio-Formats to version 6.10.0
+-  the inclusion of the experimental OMEZarrReader version 0.2.0 for reading OME-NGFF data
```
Comment (jburel) [inline history.rst]: This seems to introduce a problem when running
```
java -cp dist/lib/server/* `omero config get | awk '{print"-D"$1}'` ome.services.graphs.GraphPathReport /tmp/EveryObject.rst
Error: Could not find or load main class dist.lib.server.OMEZarrReader.jar
```

Comment (jburel) [reply history.rst]: Actually building using a non clean ``.m2`` leads to error see https://github.com/ome/openmicroscopy/pull/6324

Comment (sbesson) [reply history.rst]: I think the failures are anterior to this PR but I agree they are currently blocking the proposed set of changes for OMERO.server 5.6.5 and should be resolved before moving forward.

Comment (jburel) [reply history.rst]: They are yes.

**Comment by sbesson** [comment]
@jburel do you want me to cherry-pick https://github.com/ome/openmicroscopy/pull/6324/commits/6c5a96deeae37ea8c00d5d707f4e748a1cd3bd0d here to test the OMERO build both from an empty cache and from an existing cache populated by Ivy?

**Comment by jburel** [comment]
I was going for something more advanced see https://github.com/ome/openmicroscopy/pull/6326 but including the commit mentioned above will not hurt for now
I can sort the GHA afterwards

**Review by jburel — CHANGES_REQUESTED** [review]
omero-scripts should be bumped to 5.7.0

**Review by jburel — APPROVED** [review]

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-push#1251](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-push/1251/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-push/1251/consoleText) for more details.
Possible conflicts:
  - PR #6318 jburel 'upload assets on tag'
    - .github/workflows/source_build.yml
  - Upstream changes
    - .github/workflows/source_build.yml

--conflicts

**Pull request merged**